plugins/CorePluginsAdmin/vue/src/PasswordConfirmation/PasswordConfirmation.vue

```diff
@@ -14,13 +14,13 @@
           {{ translate('UsersManager_ConfirmThisChange') }}
         </h2>
         <h2 v-if="requiresPasswordConfirmation && !slotHasContent">
-          {{ translate('UsersManager_ConfirmWithPassword') }}
+          {{ translate('UsersManager_ConfirmWithReAuthentication') }}
         </h2>
         <div v-if="requiresPasswordConfirmation && slotHasContent">
-          {{ translate('UsersManager_ConfirmWithPassword') }}
+          {{ translate('UsersManager_ConfirmWithReAuthentication') }}
         </div>
       </div>
-      <div v-show="requiresPasswordConfirmation">
+      <div v-show="requiresPasswordConfirmation" id="password-confirmation-div">
```

Comment:
For components that are used more then once it's better to use a class instead of an id, as otherwise the id might occur more than once on a page, which is technically seen incorrect - even through browsers can handle that.